tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts

```diff
@@ -91,6 +91,7 @@ export default {
                 Ethereum: "EthereumSignature",
                 EthereumPrettified: "EthereumSignature",
                 Bitcoin: "BitcoinSignature",
+                BitcoinPrettified: "BitcoinSignature",
```

Comment:
Any strong reason to keep both?

Reply:
Analog to ethereum sig,  we keep both the standard and prettified version

Reply:
ethereum standard isn't used by IDH, so I thought of keeping the prettified version only. Enforcing prettified version for clients sounds good to me. we want all users to be able to read what clients ask them to sign.

Reply:
>  cryptic signatures (standard versions) could be abused by clients we don't control.

I don't get this one

IMO enforcing prettified version doesn't help much here, the client could still wrap it with prefix and trick users into signing it. Incidentally, the prettified version is only related to the identity verification, in trusted service invocation later it accepts standard crypto sig only.